backend/core/detection.py

```diff
@@ -872,6 +908,48 @@ def gen_implicit_withdrawal(self, monitor_event: Dict) -> NoReturn:
                     serializer="json",
                 )
 
+        def comm_annotate_hijack(self, monitor_event: Dict, hijack: Dict) -> NoReturn:
+            """
+            Annotates a hijack based on community checks (modifies "community_annotation"
+            field in-place)
+            """
+            try:
+                if hijack.get("community_annotation", "NA") in [None, "", "NA"]:
```

Comment:
why are there 3 different unset values?

Reply:
because I am not sure how each DB (postgres or sth else if we migrate) handles the default value. This is just for safety. What would be the safest, most efficient option?